File: assignment-13/qustion1.c
#include<stdio.h>
#include<stdlib.h>

int Differance(int Arr[],int iLength){
    int iCnt=0;
    int iEvenSum=0;
    int iOddSum=0;

  for ( iCnt = 0; iCnt < iLength; iCnt++)
  {
    if((Arr[iCnt]%2)==0){
        iEvenSum=iEvenSum+Arr[iCnt];
    }
    if((Arr[iCnt]%2)!=0){
        iOddSum=iOddSum+Arr[iCnt];
    }
  }
    return iEvenSum-iOddSum;
}
int main(){
    int iSize=0,iRet=0,iCnt=0;
    int *p=NULL;

    printf("Enter number of element\n");
    scanf("%d",&iSize);

    p=(int *)malloc(iSize * sizeof(int));

    if(p==NULL){
        printf("Unnable to allocate memory\n");
        return -1;
    }
    printf("Enter %d elements",iSize);
    for(iCnt=0;iCnt<iSize;iCnt++){
        printf("Enter elements %d\n",iCnt+1);
        scanf("%d",&p[iCnt]);
    }
    iRet=Differance(p,iSize);
    printf("Result is %d",iRet);

    free(p);

    return 0;
}

File: assignment-33/qustion1.java
import java.util.*;

class ArrayDemo{
    public int Arr[];

    public ArrayDemo(int x[]){
        this.Arr=x;
    }
    public int Diffrence(){
        int iCnt=0;
        int iEvenSum=0;
        int iOddSum=0;
        for(iCnt=0;iCnt<Arr.length;iCnt++){
            if((iCnt%2)==0){
                iEvenSum=iEvenSum+Arr[iCnt];
            }
            else if((iCnt%2)!=0){
                iOddSum=iOddSum+Arr[iCnt];
            }
        }
        return iOddSum-iEvenSum;
    }
}
class qustion1{
    public static void main(String args[]){
        Scanner obj=new Scanner(System.in);
        int iCnt=0;
        int iSize=0;
        int iRet=0;
        System.out.println("Enter number of element:");
        iSize=obj.nextInt();
        
        int Arr[]=new int[iSize];
        System.out.println("Enter element:");
        for(iCnt=0;iCnt<Arr.length;iCnt++){
            Arr[iCnt]=obj.nextInt();
        }

        ArrayDemo aobj=new ArrayDemo(Arr);
        iRet=aobj.Diffrence();
        System.out.println(iRet);


        
    }

}